CMakeLists.txt

```diff
@@ -35,12 +33,17 @@ option( ENABLE_DATE_TESTING "Enable unit tests" OFF )
 option( DISABLE_STRING_VIEW "Disable string view" OFF )
 option( COMPILE_WITH_C_LOCALE "define ONLY_C_LOCALE=1" OFF )
 option( BUILD_TZ_LIB "build/install of TZ library" OFF )
+option( ENABLE_DATE_INSTALL "Enable unit tests" ON )
 
 if( ENABLE_DATE_TESTING AND NOT BUILD_TZ_LIB )
     message(WARNING "Testing requested, bug BUILD_TZ_LIB not ON - forcing the latter")
     set (BUILD_TZ_LIB ON CACHE BOOL "required for testing" FORCE)
 endif( )
 
+if( ENABLE_DATE_INSTALL )
+  include( GNUInstallDirs )
+endif( )
```

Comment:
Wouldn't it make sense to move this `include()` in the installation section?

Reply:
No because we want CMAKE_INSTALL_INCLUDIR to be defined for target_include_directories

Reply:
Right, but the variable is not currently used in `target_include_directories()`; I just saw #753 though, and they seem to complement each other. Thanks!

Reply:
Yes sorry for not making it clearer, I split bug fix and feature in different pr to make review easier/faster